share/tunnel/tunnel.go

```diff
@@ -127,8 +127,19 @@ func (t *Tunnel) getSSH(ctx context.Context) ssh.Conn {
 		t.activeConnMut.RLock()
 		c := t.activeConn
 		t.activeConnMut.RUnlock()
-		return c
+	return c
+	}
+}
+
+func (t *Tunnel) getSSHNoWait(ctx context.Context) ssh.Conn {
```

Comment:
isn't `getSSH` equivalent when `c != nil` ?

Reply:
Yes you are right.
Remove the wait may also cause high cpu usage, I will revert this change.